intro-HPC/ch_scoop.tex

```diff
@@ -0,0 +1,90 @@
+\chapter{SCOOP}
+\label{ch:scoop}
+
+SCOOP (Scalable COncurrent Operations in Python) is a distributed task module allowing
+concurrent parallel programming on various environments, from heterogeneous grids to supercomputers.
+
+It can used for projects that require lots of (small) tasks to be executed.
+
+The \lstinline|myscoop| script makes it very easy to use SCOOP, even in a multi-node setup.
+
+\section{Loading the module}
+
+Before using \lstinline|myscoop|, you first need to load the \lstinline|vsc-mympirun-scoop| module. We don't specify
+a version here (this is an exception, for most other modules you should, see \autoref{subsec:explicit-version-numbers})
+because newer versions might include important bug fixes or performance improvements.
+
+\begin{prompt}
+%\shellcmd{module load vsc-mympirun-scoop}%
+\end{prompt}
+
+\section{Write a worker script}
+
+A Python worker script implements both the main program, and (typically) the small task
+function that needs to be executed a large amount of times.
+
+Using the functionality provided by the scoop module, for example the \lstinline|futures.map|
```

Comment:
`This is done using the functionality provided by the \lstinline|scoop| Python module`